pymatgen/io/cp2k/inputs.py

```diff
@@ -27,7 +27,7 @@
 import os
 import re
 import textwrap
-import warnings
+from collections import OrderedDict
```

Comment:
Normal `dict` is guaranteed to be insertion-ordered since py37. Is this needed for order-sensive equality or sth?

Reply:
That's for a very specific case. When basis sets or pseudopotentials are listed raw instead of from another file, then those keywords must maintain their order. This does happen when cp2k generates a restart file; it will put the raw potentials into those files.

I didn't know that dict are insertion ordered. Are there any issues with those dicts getting shuffled around from other operations? If that won't happen, then I agree we can probably revert to dict.

Reply:
You can overwrite dict keys but that won't change their order. The only way to change key order is to reassign the variable to a new dict (which of course can happen with `OrderedDict` too).

Reply:
Alright then yeah your changes make sense.